### PR TITLE
[doc fix and add test]fix documentation in prime and strong number

### PR DESCRIPTION
**Pull request by shellhub**
#### Description of Change

* fix doc
* add test

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C/pull/593"></a>

### CODE REVIEW

File: misc/prime.c [hunk 1]
```diff
@@ -1,35 +1,67 @@
...
+            return false;
+        }
     }
-    return 1;
+    return true;
 }
 
+/**
+ * Test function
+ */
```
Comment (kvedala) [inline misc/prime.c:41]:
```suggestion
 * @return void
 */
```
It seems that doxygen is unable to document the functions if the docstring does not contain a `@return` or `@param` identifiers. Can you add them here and for `main`.

Comment (shellhub) [reply misc/prime.c:42]: Thanks 👍

**Comment by kvedala** [comment]
Same change for the other file and also to the main function.

**Comment by shellhub** [comment]
> Same change for the other file and also to the main function.

I did. Please review again :)

**Comment by kvedala** [comment]
Change not applied to the documentation of main functions. Please add the `@return None` line to the docstring

**Comment by shellhub** [comment]
> Change not applied to the documentation of main functions. Please add the `@return None` line to the docstring

Do you mean like this ?
``` c
/**
 *Driver Code
 *@return None
*/
int main()
{
    return 0;
}
```

**Comment by kvedala** [comment]
> > Change not applied to the documentation of main functions. Please add the `@return None` line to the docstring
> 
> Do you mean like this ?
> 
> ```c
> /**
>  *Driver Code
>  *@return None
> */
> int main()
> {
>     return 0;
> }
> ```

yes 👍

**Comment by shellhub** [comment]
I did this as you say. But I don't know why need add `@return None` to main function. Because `main` return 0 to OS.  @kvedala

**Comment by kvedala** [comment]
> I did this as you say. But I don't know why need add `@return None` to main function. Because `main` return 0 to OS. @kvedala

it is only for the documentation generator. The generator ignored the functions if the doctoring does not contain a `@return` or `@param` tags. Please correct that to `0` for main as needed. Thanks, good catch.

**Comment by shellhub** [comment]
Can I merge this branch?

**Pull request merged**